tweaks/steam.yaml

```diff
@@ -1318,6 +1357,9 @@
 "798790": # PolyCube
   compat_tool: proton_513
 
+"349730": Popup Dungeon
```

Comment:
missing '#' :sweat_smile:

Reply:
Done. These hashes are driving me crazy.
Any chance that we create a test automate who validates the entries?

Reply:
Yeah, I have been thinking about that as well.

Reply:
I usually run this simple python script, but it doesn't catch everything
```
import yaml
data = yaml.load(open('./tweaks/steam.yaml'), Loader=yaml.FullLoader)
```